tests/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@taqueria/test",
+  "name": "@taqueria/tests",
```

Comment:
Can we add a `"private":  true` to the package.json please? This will ensure that this package never accidentally gets published. Also, helps communicate that to others looking here.
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Reply:
I have updated this to be private